changelogs/current.yaml

```diff
@@ -50,6 +50,49 @@ new_features:
 - area: dns_resolver
   change: |
     added DNS stats for c-ares DNS resolver. Detailed documentation is available :ref:`here <arch_overview_dns_resolution>`.
+    added :ref:`include_unroutable_families<envoy_v3_api_field_extensions.network.dns_resolver.apple.v3.AppleDnsResolverConfig.include_unroutable_families>`
```

Comment:
Looks like you picked up some extra release note changes. Bad merge? Please resolve.

Reply:
Done, thanks.